pkgs/applications/misc/nwg-displays/default.nix

```
@@ -38,6 +40,8 @@ python310Packages.buildPythonApplication rec {
     python310Packages.gst-python
     python310Packages.i3ipc
     python310Packages.pygobject3
+  ] ++ lib.optional hyprlandSupport [
```

`lib.optional` already wrap in an list, so if we keep this, it would wrap `wlr-randr` twice.

Use `lib.optionals` for avoiding that issue.

Pro tip: it's better to use `lib.optionals` all the time in the future.

```suggestion
  ] ++ lib.optionals hyprlandSupport [
```

---

I actually intended to use `lib.optionals` initially but got them confused :D 
I did notice the mistake afterwards, but didn't bother changing since it passed the review.

Somewhat counter-intuitively, [this actually works](https://github.com/NixOS/nixpkgs/commit/bbb813279055ee2563054d839ff7a2ee696a9ce7#r39015214). Of course, it's not very semantically clear.